support/modules/docker

```diff
@@ -18,17 +18,28 @@ case "$STATE" in
         ;;
 
     ArtifactInstall)
+
+        command -v docker >/dev/null 2>&1 || { echo >&2 "ERROR: Must have docker installed to run this update module."; return 1; }
```

Comment:
I would recommend doing this check in `Download` instead. The reason is that if `ArtifactInstall` fails, it will try to run `ArtifactRollback`, which will probably also fail, resulting in the `INCONSISTENT` label. `Download` however, does not change the system, and hence the update immediately stops after a failure there.

Reply:
Yes, great comment! 
Will do the check in the downloads state

Reply:
Done